projects/novo-elements/src/elements/form/extras/file/FileInput.spec.ts

```diff
@@ -225,16 +226,40 @@ describe('Elements: NovoFileInputElement', () => {
   //     });
   // });
   //
-  // describe('Method: writeValue()', () => {
-  //     it('should be defined.', () => {
-  //         expect(component.writeValue).toBeDefined();
-  //     });
-  //
-  //     it('should change the value', () => {
-  //         component.writeValue(10);
-  //         expect(component.model).toBe(10);
-  //     });
-  // });
+  describe('Method: writeValue()', () => {
+      it('should be defined.', () => {
+          expect(component.writeValue).toBeDefined();
+      });
+
+      it('should change the value', () => {
+        component.writeValue(10);
+        expect(component.model).toBe(10);
+      });
+  
+      it('should empty the file list if a falsey value is programmatically set', () => {
+          component.writeValue(undefined);
+          expect(component.files).toEqual([]);
+      });
+  });
+  describe('Method: check(files)', () => {
+    beforeEach(() => {
+      component.layoutOptions = {};
+    });
+    it('should be defined.', () => {
+        expect(component.writeValue).toBeDefined();
```

Comment:
Did you mean to check this method to be defined?

Reply:
Not particularly, I uncomments those tests and added my own. I have removed the function definition tests.